src/mozxchannel/git/pull_request.py

```diff
@@ -0,0 +1,19 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import os
+import subprocess
+
+
+def create(target, *, title, message, branch=None):
```

Comment:
Are there plans to move this to treescript or other scriptworker at some point?

Reply:
No, I don't have any plans to do that. This is pushing as an unprivileged user, so I think taskcluster scopes are enough protection.